### ISSUE TITLE
Module not working in Android 10

### ISSUE THREAD

**Issue by amitcoolcampus** [YaDev/SamsungDualSimFix#1]
I tried using this module in Samsung note 9, Android 10. But after flashing my note 9 is not coming up. It is in bootloop.
Although it comes up on the start screen but immediately in 5 seconds. Even before I enter my password to unlock phone it restarts. 

Can you please fix it, i really need it.

**Comment by amitcoolcampus** [comment]
Can you please look into this?

**Comment by YaDev** [comment]
I will take a look at the new services package to fix it.

**Comment by amitcoolcampus** [comment]
Thank you so much. I'll keep an eye.

**Comment by amitcoolcampus** [comment]
Hello Brother, any update on this.

**Comment by YaDev** [comment]
Can you upload these files from your device ?

system/framework/services.jar 
system/framework/oat/arm64/services.odex
system/framework/oat/arm64/services.vdex

**Comment by amitcoolcampus** [comment]
[https://drive.google.com/file/d/19w9Em-gzbYhOq_3TR7OxE60vPg6I0cEm/view?usp=drivesdk](url)

**Comment by amitcoolcampus** [comment]
Sure above link you can download all of them.

**Comment by YaDev** [comment]
> 
> 
> Sure above link you can download all of them.

Needs permissions

**Comment by amitcoolcampus** [comment]
Extremely sorry. Please check now.

**Comment by YaDev** [comment]
I have updated the module.
Can you try this:

https://gofile.io/d/gOotxq

**Comment by amitcoolcampus** [comment]
> I have updated the module.
> Can you try this:
> 
> https://gofile.io/d/gOotxq

Thank you so much. :) 

I can't do a full fledged testing right now as I am not carrying hybrid sim adapter. It's at my another home and Due to Lockdown, it's not possible for me to go home. However I have just ordered a new one at current address to test this and I am expecting to receive it in 1-2 days. Post that I will update you about the results.

For the time being, I tried to install this module and good news is My phone was able to boot up successfully unlike the previous version. It got installed smoothly and my phone is working with the module activated. So I am hoping functionality will also work fine.

**Comment by ferry77777** [comment]
Hi Bro.. i did try install your apk for dualsim and sd card active, but still not working, please check my screenshoot. Hopefully you can help us to solve the problem.
![Screenshot_20200517-170621_Settings](https://user-images.githubusercontent.com/65489054/82141553-d1a8bb80-9860-11ea-893d-15d6bb0c2efc.jpg)
![Screenshot_20200517-165909_Chrome](https://user-images.githubusercontent.com/65489054/82141555-d40b1580-9860-11ea-93f2-7e98ef53db31.jpg)

**Comment by YaDev** [comment]
Can you post the log file form EdXposed Manager ?

Also, I need you more info about your device and which version of EdXposed you have.

**Comment by amitcoolcampus** [comment]
Hi Dev,

I just received my hybrid sim slot adapter. I tested this module and it's working fine for me. 
I would like to thank you for all your hard work to make it work. God bless you. 
Kudos!

**Comment by YaDev** [comment]
Thanks
I have updated the module.

**Comment by amitcoolcampus** [comment]
Hi YaDev, I got the error again. Not sure if I am right, but I got this error when I cleared cache and dalvik memory and rebooted. I think error popped up before edxposed could load so I got the error and once it appeared there was no solution but to restart the phone. However I could notice behind the error, network was there for both the sims. 
![Screenshot_20200611-141029_Nova Launcher](https://user-images.githubusercontent.com/13386188/84817795-2ba0ca80-b033-11ea-8c7a-b816526db719.png)
